Month1/Week2/Section1/Exercise2/Main2.cpp

```diff
@@ -0,0 +1,73 @@
+
+
+
+/*Exercise 2 - Military Clock
+Create a class called MilitaryClock that stores and displays the military time.
+
+Your class should contain the following constructors and methods:
+MilitaryClock();
+MilitaryClock(short hours, short minutes);
+void SetTime(short hours, short minutes);
+void DisplayTime();
+Example output
+0000
+0700
+1215
+*/
+#include <iostream>
+
+#include<string>
+
+class MilitaryClock
+{
+private:
+	short hours ;
+	short minutes ;
+public:
+	std::string Texto;
+	MilitaryClock()
+	{
+		 hours=0;
+		 minutes=0;
+	}
+	MilitaryClock(short hour, short minute)
+	{
+		 hours = hour;
+		 minutes = minute;
+	}
+	void SetTime(short sethours, short setminutes)
```

Comment:
What's the point of requiring parameters if you're going to change their values to the same literals every time? This function is not doing what it's supposed to do.